### ISSUE TITLE
Record fetch performance in Google Analytics

### ISSUE THREAD

**Issue by bobsilverberg** [mozilla/addons-code-manager#1450]
This is a spin-off from #1423.

This is about capturing `fetch` performance via `window.Performance` and recording it in Google Analytics so that we can report on it and identify performance bottlenecks.

For QA:

After this lands, "beacons" for Google Analytics pings should appear in the network panel to indicate that a timing was sent for each `fetch` operation. There will also be a console message for each one. The console messages will look like:

`[GA: ON] Sending timing for {"category":"resource","label":"http://localhost:3000/api/v5/accounts/profile/?lang=en-US","value":298,"variable":"fetch"}`

The network panel entries will look like:

<img width="1460" alt="Screenshot 2020-02-24 14 42 40" src="https://user-images.githubusercontent.com/142755/75185424-11558080-5714-11ea-8e5c-3aafb76bd50d.png">

and if you click on the network panel entry for the "beacon", and check the "Params" tab, you'll see the content of the beacon, which, for the example quoted above, would look like `v=1&_v=j81d&a=132045738&t=timing&_s=1&dl=http%3A%2F%2Flocalhost%2Fen-US%2Fbrowse%2F502955%2Fversions%2F1541786%2F&ul=en-us&de=UTF-8&dt=Addons%20Code%20Manager&sd=24-bit&sr=960x600&vp=1455x419&je=0&fl=32.0%20r0&utc=resource&utv=fetch&utl=http%3A%2F%2Flocalhost%3A3000%2Fapi%2Fv5%2Faccounts%2Fprofile%2F%3Flang%3Den-US&utt=298&_u=SACAAAAB~&jid=1792451120&gjid=1478828136&cid=474789440.1575918409&tid=UA-77033033-31&_gid=1676997539.1582562163&_r=1&z=1512349024`

Note that the value of `utl` above is the same as the `label` in the console message, and that the value of `utt` is the same as the `value` in the console message.

**Comment by bobsilverberg** [comment]
It looks like this is working and these are now being captured on dev.

**Comment by ioanarusiczki** [comment]
@bobsilverberg Looks good to me. I checked on dev and I can confirm the GA events are registered as described [above](https://github.com/mozilla/addons-code-manager/issues/1450#issue-568328415). 

There's only one behavior I'd like to ask about because I can't tell if it is an expected behavior:
- if I navigate on a couple of files (let's say I am on a Browse page) and I open another Browse/Compare page, then I return back to the initial one - the viewed files I clicked on will no longer show a 200/304 (I expected a 304) in the Network and therefore no GA `beacon` will be sent either. Obviously, it's the same if I click some files on a Browse/Compare page and then I simply return to the ones I clicked on. 

An example: 

![no 304_01](https://user-images.githubusercontent.com/33448286/75671876-3bfd8700-5c88-11ea-80cb-8e6970fe97a2.gif)
